tests/integration-tests/conftest.py

```diff
@@ -521,6 +530,14 @@ def setup_sts_credentials(region, request):
     unset_credentials()
 
 
+@pytest.fixture(scope="class", autouse=True)
+def setup_env_variable(region):
```

Comment:
can't we have those fixtures depend on the region one so that we don't rely on this env variable. For example when creating an object of type Cluster we could pass the region and then use the `--region` argument when executing commands.

My concern is that by setting the env variable we create an implicit dependency. During the teardown of fixtures since Cluster does not explicitly depend on region we could have this fixture being deleted before Cluster and thus breaking the cleanup of the Cluster fixture

Reply:
The env var is only for cluster commands. Here `scope="class", autouse=True` guarantees the correct execution order with `clusters_factory` which has `scope="class"`. It is also feasible to add --region to each of commands needed. For now, I was wondering if I could just use this minimal change and add a FixMe comment?

Reply:
I agree that Cluster is not affected now. I'm afraid that future fixtures will start to rely on this without being aware of this side effect. ok with marking as fixme and handling separately